relevanceai/operations/cluster/algorithms/community_detection.py

```diff
@@ -4,8 +4,16 @@
 
 
 class CommunityDetection:
-    def __init__(self, gpu=False):
+    def __init__(self, config, gpu=False):
+
         self.gpu = gpu
+        self.threshold = 0.75
```

Comment:
hard-coded values should be at the top - never set as an attribute later on as this should be easily adjusted.

Reply:
true